app/src/sharedTest/java/org/oppia/app/player/state/StateFragmentTest.kt

```diff
@@ -803,6 +805,20 @@ class StateFragmentTest {
     }
   }
 
+  @Test
+  fun testStateFragment_loadRatioInput_checkRatioAnswer() {
```

Comment:
```suggestion
  fun testStateFragment_inputRatio_submit_correctAnswerDisplayed() {
```

Alternative wording that's a bit clearer on the outcome/conditions.

Reply:
done